portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/CPRolesPGStaging.testcase

```diff
@@ -39,251 +47,281 @@ definition {
 	test AccessStaging {
 		property portal.acceptance = "true";
 
-		JSONStaging.enableLocalStaging(groupName = "Site Name");
+		task ("Given: a regular user is added") {
+			JSONStaging.enableLocalStaging(groupName = "Site Name");
 
-		JSONUser.addUser(
-			userEmailAddress = "userea@liferay.com",
-			userFirstName = "userfn",
-			userLastName = "userln",
-			userScreenName = "usersn");
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+		}
 
-		JSONRole.addRegularRole(
-			roleKey = "Regrole Name",
-			roleTitle = "Regrole Name");
+		task ("And: A regular role is created") {
+			JSONRole.addRegularRole(
+				roleKey = "Regrole Name",
+				roleTitle = "Regrole Name");
 
-		Role.openRolesAdmin();
+			Role.openRolesAdmin();
```

Comment:
Please, move this macro to the next task (task ( "When: the user is assigned to regular role") )